symbol sets/src/ISOM2017_15000.xmap

```diff
@@ -0,0 +1,10029 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<map xmlns="http://openorienteering.org/apps/mapper/xml/v2" version="6">
+    <notes>ISOM 2017
+Symbols for OpenOrienteering Mapper
+Created by Jukka Liikari and Håvard Tveite, IOF Map Commission
+</notes>
+    <georeferencing scale="15000">
+        <projected_crs id="Local"/>
+    </georeferencing>
+    <colors count="27">
+        <color priority="0" name="Purple" c="0.150" m="1.000" y="0.000" k="0.000" opacity="1.000">
+            <spotcolors>
+                <namedcolor>Purple</namedcolor>
+            </spotcolors>
+            <cmyk method="custom"/>
+            <rgb method="custom" r="0.850" g="0.000" b="1.000"/>
+        </color>
+        <color priority="1" name="White background" c="0.000" m="0.000" y="0.000" k="0.000" opacity="1.000">
+            <cmyk method="custom"/>
+            <rgb method="custom" r="1.000" g="1.000" b="1.000"/>
+        </color>
+        <color priority="2" name="All color separations" c="1.000" m="1.000" y="1.000" k="1.000" opacity="1.000">
```

Comment:
Ok. I can see two ways from here.
The first one is:
1. We are asking for permission from @havatv and Jukka Liikari to distribute file isom-201x-finaldraft-11-2015-inofficialsymbols.omap under GNU/GPL v3 as a part of OpenOrienteering Project. And if they do not mind
2. I'm creating new pull request with this file + my commits
3. OpenOrienteering community is working and polishing this file

The second way is:
1. OpenOrienteering community is creating new symbol set, based on existing ISOM2000.

I don't really like the second way, because a lot of work already done. Many symbols was totally created from scratch, but licensing collision means that I can't just copy-paste them from one file to another.

![dsc_0620-1](https://cloud.githubusercontent.com/assets/10380431/25312234/fd4cff2e-281c-11e7-9ce1-aa65cc480e95.jpg)

@dg0yt please approve one of the options before I do something else unnecessary

@999999333 - For your information